src/main/java/me/saharnooby/plugins/leadwires/chunk/LoadedChunkTracker.java

```diff
@@ -85,7 +85,13 @@ public void onPacketSending(PacketEvent e) {
 			} else if (type == PacketType.Play.Server.RESPAWN) {
 				set.clear();
 			} else {
-				ChunkCoord coord = new ChunkCoord(packet.getIntegers().read(0), packet.getIntegers().read(1));
+                ChunkCoord coord;
+                if (NMSUtil.getMinorVersion() >= 20 && NMSUtil.getReleaseVersion() > 1) {
+                    coord = new ChunkCoord(packet.getStructures().read(0).getIntegers().read(0), packet.getStructures().read(0).getIntegers().read(1));
+
+                } else {
+                    coord = new ChunkCoord(packet.getIntegers().read(0), packet.getIntegers().read(1));
+                }
```

Comment:
Please fix the indentation on this and the 5 lines above.

Reply:
thats crazy, in my ide the indentation is correct :D 
![image](https://github.com/saharNooby/lead-wires/assets/76794156/143e21e5-e8ae-4120-82e6-14272475a15b)

Reply:
It's tabs vs spaces :)